### ISSUE TITLE
64 bit script uses wrong shell in Debian

### ISSUE THREAD

**Issue by keantoken** [jhernberg/wineasio#3]
Hello. in Debian the prepare_64bit_asio script should start with

#!/bin/bash

As the shell it normally refers to does not support the <<< operator.

Otherwise it fails with a syntax error.

**Comment by osirisgothra** [comment]
Yeah noticed this too. POSIX sh does not support a '<<<' redirection, that is at least a bash thing, though it works also in zsh and probably other shells too. I can't remember the last distro I had that *didnt* include bash (and often as the default shell), so I doubt using bash would be a problem.
